FashionConnect_api/routes/users.js

```diff
@@ -195,4 +193,79 @@ router.delete('/collections/:collectionID', async (req, res) => {
   }
 });
 
+//Route to handle forgot password
+router.post('/forgotpassword/', async (req, res) => {
+
+  try {
+    const { email, token } = req.body;
+    // Find the existing user in the database
+    const user = await User.findOne({ where: { email: email } });
+
+    if (!user) {
+      return res.status(404).json({ error: 'User not found' });
+    }
+
+    const userID = user.id
+
+    // Create a new ForgotPassword in the database
+    const passwordAUTH = await ForgotPassword.create({
+      userID: userID,
+      token: token
+    });
+    const receiverEmail = email;
+    const subject = "FORGOT-PASSWORD TOKEN";
+    const text = `Here is your token ${token}`;
+
+    sendEmail(receiverEmail, subject, text);
+
+    return res.status(201).json({ message: 'Collection deleted successfully', passwordAUTH: passwordAUTH });
+  } catch (error) {
+    return res.status(500).json({ error: 'Failed to delete collection' });
+  }
+});
+
+//Route to confirm token 
+router.post('/reset-password/', async (req, res) => {
+
+  try {
+    const { token } = req.body;
+    // Find the existing userid in the token database
+    const record = await ForgotPassword.findOne({ where: { token: token } });
+
+    if (!record) {
+      return res.status(404).json({ error: 'Inavalid Token' });
+    }
+
+    const userID = record.userID
+
+    const activeUser = await User.findOne({ where: { id: userID } })
+
+    return res.status(201).json({ message: 'Collection deleted successfully', user: activeUser });
```

Comment:
These messages seem to be copied from the Collections feature. Can you change them to the right ones for password recovery.